src/jobs/test-package.yml

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2018, salesforce.com, inc.
+# All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+# For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+
+description: >
+  Runs tests for package
+
+parameters:
+  node_version:
+    description: version of node to run tests against
+    type: string
+    default: latest
+
+executor:
+  name: default
+  tag: << parameters.node_version >>
+
+steps:
+  - checkout
+  - restore_cache:
+      keys:
+        - v1-npm-{{checksum ".circleci/config.yml"}}-{{checksum "yarn.lock"}}
```

Comment:
You allow them to specify the package manager in the publish but not in these scripts. Is that something we want to support?

Reply:
@amphro hm, that's a good point. Now that I'm thinking about it, I think we should force people into using `yarn`. We can support `npm` down the road if we need to

Reply:
Sounds good to me.